src/Employer/Employer.Web/Views/Part1/Location/Location.cshtml

```diff
@@ -1,68 +1,97 @@
 @using Esfa.Recruit.Employer.Web.ViewModels.Part1.Location;
-@using Esfa.Recruit.Shared.Web.Mappers;
+@using Esfa.Recruit.Shared.Web.Configuration
+@using Microsoft.Extensions.Options
+@inject IOptions<PostcodeAnywhereConfiguration> PcaConfig
+
 @model LocationViewModel;
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
         <a asp-show="@Model.CanShowBackLink" asp-route="@RouteNames.EmployerName_Get" esfa-automation="link-back" class="govuk-back-link govuk-!-margin-top-0 govuk-!-margin-bottom-6">Back</a>
-        <partial name="@PartialNames.ValidationSummary" model='new ValidationSummaryViewModel { ModelState = ViewData.ModelState }'/>
+        <partial name="@PartialNames.ValidationSummary" model='new ValidationSummaryViewModel { ModelState = ViewData.ModelState }' />
         <h1 class="govuk-heading-xl govuk-!-margin-bottom-6">Where will the apprentice work?</h1>
-
         <form asp-route="@RouteNames.Location_Post" asp-route-wizard="@Model.PageInfo.IsWizard">
             <div class="govuk-form-group">
                 <fieldset class="govuk-fieldset">
-                    <span esfa-validation-message-for="UseOtherLocation" class="govuk-error-message"></span>
-                    <div class="govuk-radios">
-                        <div class="govuk-radios__item">
-                            <input asp-for="UseOtherLocation" value="false" id="legal-entity-name" class="govuk-radios__input" type="radio" >
-                            <label for="legal-entity-name" class="govuk-label govuk-radios__label">
-                                    @Model.LegalEntityLocation
-                            </label>
-                        </div>
-<!--                        
-                        <div class="govuk-radios__item">
-                            <input asp-for="UseOtherLocation" value="true" id="legal-entity-name" class="govuk-radios__input" type="radio" >
-                            <label for="legal-entity-name" class="govuk-label govuk-radios__label">
-                                    somewhere else
-                            </label>
+                    <div class="govuk-radios govuk-radios--conditional" data-module="radios">
+                        <span esfa-validation-message-for="Location" class="govuk-error-message"></span>
+                        <div class="govuk-radios">
+                            <div class="govuk-radios__item">
+                                <input asp-for="Location" value="LegalEntityLocation" id="legal-entity-id" class="govuk-radios__input" type="radio">
+                                <label for="legal-entity-name" class="govuk-label govuk-radios__label">
+                                    @Model.LocationName
+                                </label>
+                            </div>
+                            @if (Model.OtherLocationsAddress != null && Model.OtherLocationsAddress.Any())
```

Comment:
We can get rid of this `IF` when we set OtherLocationsAddress to an empty list as commented elsewhere.